metrics/examples/prometheus/install/prometheus-rules.yaml

```diff
@@ -48,7 +56,46 @@ spec:
         severity: warning
       annotations:
         summary: 'Kafka offline log directories'
-        description: 'There are {{ $value }} offline log directoris on {{ $labels.kubernetes_pod_name }}'
+        description: 'There are {{ $value }} offline log directories on {{ $labels.kubernetes_pod_name }}'
+    - alert: ScrapeProblem
+      expr: up{job="kubernetes-services",kubernetes_namespace!~"openshift-.+"} == 0
```

Comment:
Wouldn't this fire for many Kafka unrelated jobs if they have any Prometheus issues? Or how does this select only the Strimzi pods?